terraform/github-zephyrproject-rtos/team/team-members/infrastructure.csv

```diff
@@ -1,4 +1,5 @@
 username,role
+fabiobaltieri,member
 galak,member
```

Comment:
looks like some cleanup would be in order :)

Reply:
yeah in the tsc file as well, was about to do it but figured it may deserve a separate one going over other groups too